src/tox/session/cmd/run/common.py

```diff
@@ -33,7 +33,7 @@ def __call__(
     ) -> None:
         value = "true" if values is None else values
         if value not in ("config", "true", "false"):
-            raise ArgumentError(self, f"value must be 'config', 'true', or 'false' (got {repr(value)})")
+            raise ArgumentError(self, f"value must be 'config', 'true', or 'false' (got {value!r})")
```

Comment:
Why is the new version better than the old one?

Reply:
It's more about formatting a string than about calling some external function.

Reply:
I'm ok with the new version.